spec/requests/cloud_volumes_spec.rb

```diff
@@ -100,6 +100,22 @@
     expect(response).to have_http_status(:not_found)
   end
 
+  it 'can safe delete cloud volumes' do
+    ems = FactoryBot.create(:ems_autosde, :name => "Autosde")
+    volume = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => ems, :name => "my_volume")
+    api_basic_authorize(action_identifier(:cloud_volumes, :safe_delete, :resource_actions, :post))
+
+    post(api_cloud_volume_url(nil, volume), :params => {"action" => "safe_delete"})
+
+    expected = {
+      "success" => true,
+      "message" => "Deleting Cloud Volume my_volume"
+    }
+
+    expect(response.parsed_body).to include(expected)
+    expect(response).to have_http_status(:ok)
+  end
+
```

Comment:
@gregoryb-at-ibm could you add a spec trying to safe_delete a non-autosde cloud volume and expect to get the "unsupported" error back as well?

Reply:
Added. I'll run another cross repo test

Reply:
That's alright core and provider are merged and a new test won't impact the UI PR